ring/lifecycler.go

```diff
@@ -522,6 +523,15 @@ func (i *Lifecycler) ZonesCount() int {
 	return i.zonesCount
 }
 
+// Zones return the list of zones for which there's at least 1 instance registered
+// in the ring.
```

Comment:
[nit] Make it clear that zones are guaranteed to be sorted, so the output is stable.

Reply:
It now reads like this:

> // Zones return the list of zones for which there's at least 1 instance registered
> // in the ring. They are guaranteed to be sorted alphabetically.